src/consumption.cpp

```diff
@@ -52,6 +55,72 @@ int player::nutrition_for( const itype *comest ) const
     return ( comest && comest->comestible ) ? comest->comestible->nutr : 0;
 }
 
+std::map<vitamin_id, int> player::vitamins_from( const itype_id &id ) const
+{
+    return vitamins_from( item( item::find_type( id ) ) );
```

Comment:
Why call `find_type` here, doesn't the `item` constructor work with a plain id as well?